extensions/wikia/VideosModule/Modules/VideosModuleBase.class.php

```diff
@@ -122,7 +122,6 @@ public function getVideos() {
 	protected function getCacheKey() {
 		$cacheKey = wfSharedMemcKey(
 			self::CACHE_VERSION,
-			__CLASS__,
```

Comment:
This wasn't specifically causing a problem, but __CLASS__ was always referring to the base class here. We could use get_called_class() to get the inherited class, but this->source conveys the same information.